css-viewport/Overview.bs

```diff
@@ -356,6 +356,154 @@ only the value previously set to it.
+computed values that are ''auto'' or &lt;percent&gt;.
+
+<div class="note">Unlike 'transform', scaling the 'zoom' property affects
+layout.</div>
+
+<p class="note"> The computed value of 'font-size' is never &lt;percent&gt;
+thus 'zoom' always applies.</p>
+
+<p class="note">'zoom' does not affect or prevent 'transform' scaling.</p>
+
+<pre class="propdef">
+Name: zoom
+Value: &lt;number&gt;
+Initial: 1
+Applies to: all &lt;length&gt; property values of block-level elements
+Inherited: no
+Percentages: Refer to all &lt;length&gt; property values
+Media: visual
+Computed value: as specified
+</pre>
+
+The values of this property have the following meanings:
+
+<dl dfn-for="wrap-flow" dfn-type=value>
+  <dt><dfn lt="'zoom'!!'number'">number</dfn>
+  <dd>
+    Positive floating point number indicating a zoom factor. Numbers smaller
+    than 1.0 indicate a "zoom out" or minification effect, while numbers
+    greater than 1.0 indicate a "zoom in" or magnification effect.
+
+  <dt><dfn lt="'zoom'!!&lt;percent&gt;">percent</dfn>
+  <dd>
+    Positive floating point number, followed by a percentage character ("%")
+    which indicates a zoom factor multiplied by 100.
+</dl>
+
+Negative or zero values for 'zoom' are illegal.
+
+<div class="example">
+  Example of the 'zoom' property applied during hover for magnification effect.
+
+<pre>
+&lt;div class="messageBox"&gt;
+    &lt;div class="label"&gt;Text of the label&lt;/div&gt;
+&lt;/div&gt;
+
+&lt;style type="text/css"&gt;
+.messageBox {
+    width: 10em;
+    padding: 2em;
+    border: medium solid lightblue;
+}
+
+.messageBox:hover {
+    zoom: 150%;
+}
+
+.label {
+    background: lightgrey;
+    padding: 1em;
+    text-align: center;
+}
+&lt;/style&gt;
+</pre>
+
+  Illustration of the before and after hover state of the message box element.
+  <img src="css_zoom_hover_example.png" />
+</div>
+
+<div class="example">
+Example of nested zoom. In this example, "Inner text" is 4x as large as
+"Outer text", and "Middle text" is 2x as large as "Outer text".
+
+<pre>
+&lt;div style="zoom: 2"&gt;
+  Middle text
+	&lt;div style="zoom: 2"&gt;
+	   Inner text
+	&lt;div&gt;
+&lt;div&gt;
+Outer text
+</pre>	 
+</div>
+
+<div class="example">
+Example of replaced elements. In this example, the image and iframe will
+be twice as large as their default sizing.
+
+<pre>
+&lt;div style="zoom: 2"&gt;
+	&lt;img src=""&gt;
+	&lt;iframe src=""&gt;
+&lt;div&gt;
+</pre>	 
+</div>
+
+The <dfn>effective zoom</dfn> of an element is the product of its computed
+value of 'zoom' and all ancestors' computed values of 'zoom'.
+
+The <dfn>scaled</dfn> value of a CSS length is the used value of that length;
+in particular it includes zoom.
+
+The <dfn>unscaled</dfn> value of a CSS length relative to an element is the
+<a>scaled</a> value divided by the element's <a>effective zoom</a>.
+
+<div class="note">Zoom applied to an iframe multiplies all its CSS lengths
```

Comment:
If it's a one-paragraph note, you can just prepend it with "Note: " and Bikeshed will handle the markup.